spec/avromatic/model/raw_serialization_spec.rb

```diff
@@ -26,47 +27,125 @@
       expect(instance.value_attributes_for_avro).to eq(expected)
     end
 
+    context "with a nested record" do
+      let!(:test_class) do
```

Comment:
You're right. I didn't realize I could access nested_models directly via a Model.nested_models. Because I was using Avromatic.nested_models, I need let! for force the side effect. I'll fix it here and throughout.